docs/delete-kafka-operator.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Delete Kafka operator
+shorttitle: Delete operator
+weight: 950
+---
+
+In case you want to delete the Kafka operator from your cluster, note that because of dependencies between the various components, they must be deleted in specific order.
+
+{{< warning >}}It’s important to delete the kafka-operator deployment as the last step.
+{{< /warning >}}
+
+1. First delete all *KafkaUser* and *KafkaTopic* custom resources. The *kafka-operator* watches these types of resources, and when these are deleted, it removes the appropriate resources from the Kafka cluster.
```

Comment:
This one is not necessary needed since the operator tries to delete all of the related User and Topic once the Cluster is deleted.